Assignment2b

toasterdoom opened the pull request.

paoespinozarias commented on Assignment2/toasterdoom-fromJSONtoJSON-LD.jsonld:
```diff
@@ -0,0 +1,50 @@
+{
+    "@context": {
+        "student": "http://fi.upm.es/alumnos/b.agirre",
+        "subject": "http://fi.upm.es/GII/asignaturas"
```
Add / or # at the end of URI definitions.

paoespinozarias commented on Assignment2/toasterdoom-fromJSONtoJSON-LD.jsonld:
```diff
@@ -0,0 +1,50 @@
+        "student": "http://fi.upm.es/alumnos/b.agirre",
+        "subject": "http://fi.upm.es/GII/asignaturas"
+    },
+    "student:@id": "b.agirre",
+    "student:@type": "Student",
```
"@id" and "@type" are JSON-LD tags, thus remove the prefix. 
You have to add the prefix at the id and type values. e.g. "student:bagirre"

paoespinozarias commented on Assignment2/toasterdoom-fromRDFtoJSON-LD.jsonld:
```diff
@@ -0,0 +1,34 @@
+{
+    "@context": {
+        "base": "http://assignment2.org/#",
```
you don't need to define this base prefix.

toasterdoom replied:
Thanks for the feedback! It's fixed now

toasterdoom commented:
I updated my fork and then reviewed your changes. But when I tried to do a pull request it said 600 files were deleted, something was wrong. I updated my fork to upstream somehow, but it automatically closed this pull request. I also reverted all my Assignment2 changes and my Assignment 3; thankfully I made a backup. To be honest, I don't know what I was doing.
I will now create another pull request with the Assignment2b fixes and Assignment3
